deploy/main.tf

```
@@ -43,8 +43,21 @@ chmod 0600 /root/nixbuild.pem
 EOF
 }
 
-output "server_ip_addr" {
-  value = aws_instance.binplz_server.public_ip
+output "public_ip_addr" {
+  value = aws_eip.binplz_eip.public_ip
+}
+
+resource "aws_eip" "binplz_eip" {
+  instance = aws_instance.binplz_server.id
+}
+
+resource "null_resource" "dns_update" {
+  triggers = {
+    ip_change = aws_eip.binplz_eip.public_ip
```

Thought it might be nice to have a little note that we don't actually expect to use the `deploy/update_dns.py` script.

```suggestion
    # Note that after deploying binplz at least once, we will likely never re-provision this Elastic IP, so it is very unlikely to ever change. 
    ip_change = aws_eip.binplz_eip.public_ip
```